src/scripts/pypa-build.sh

```diff
@@ -0,0 +1,14 @@
+main() {
+  local -a build_args
+  build_args=(
+    --outdir "$PARAM_OUTDIR"
+  )
+  [[ $PARAM_SDIST ]] && build_args+=( --sdist )
+  [[ $PARAM_WHEEL ]] && build_args+=( --wheel )
+  [[ $PARAM_SKIP_DEPENDENCY_CHECK ]] && build_args+=( --skip-dependency-check )
+  [[ $PARAM_NO_ISOLATION ]] && build_args+=( --no-isolation )
```

Comment:
All these parameters seem to be false by default. Should we write one or two tests for when some of them are true?